data-model/src/main/scala/za/co/absa/enceladus/model/versionedModel/VersionedSummary.scala

```diff
@@ -16,3 +16,5 @@
 package za.co.absa.enceladus.model.versionedModel
 
 case class VersionedSummary(_id: String, latestVersion: Int)
+
+case class VersionList(_id: String, versions: Seq[Int])
```

Comment:
In theory, it could. Or even `maxVersion` would do (assuming that all entities start with v1).

Attachments are the exception, they take after the schema version and there may be holes, but attachments are not exposed directly.

Then there is the matter of not being able to prove that all versions are actually there in the DB, we assume they should be, but... how sure can you be. 

I would be hesitant to provide such guarantees right of the bat.

Reply:
Perhaps let's just use just VersionedSummary - with the latest version

Reply:
Done.